content/articles/how-to-build-a-task-manager-with-angular/index.md

```diff
@@ -0,0 +1,468 @@
+# How to build a task manager with angular.
+
+## Introduction
+
+If you are trying to learn a programming language or framework, it is good practice to build simple tasks which will go a long way to help solidify your knowledge in that language.  
+
+In this article, I will work you through building a simple task manager app (popularly known as a Todo app) using [angular](https://angular.io/docs) a JavaScript framework.
+
+
+## What is a Task Manager?
+
+A task manager is used to write, organize and rearrange tasks more efficiently. The task manager we are going to build will allow the user to add a task, mark the task as complete and also delete the task.
+
+
+## Key Takeaways
+1. How to create and structure an angular project.
+2. How to create modules and components in angular.
+3. Build a simple application.
+4. Save tasks to local storage
+
+
+
+## Prerequisite
+
+Before we get started, here are a few things to note:
+
+- You should have [Node.js](https://nodejs.org/en/) installed on your computer.
+- You should be fairly familiar with [Angular](https://angular.io/docs) and JavaScript.
+
+Knowledge of [TypeScript](https://www.typescriptlang.org/) is useful but not requisite.
```

Comment:
Remove this line, and, condense it on the same line with the one I pointed above.